### DISCUSSION TITLE
dose horovod-docker support tensorflow 1.x

### DISCUSSION THREAD

**Discussion by chenxinhua** [horovod/horovod#2797] [Q&A]
**Environment:**
1. Framework: (TensorFlow, Keras, PyTorch, MXNet)
2. Framework version:
3. Horovod version:
4. MPI version:
5. CUDA version:
6. NCCL version:
7. Python version:
8. OS and version:
9. GCC version:

**Checklist:**
1. Did you search issues to find if somebody asked this question before?
2. If your question is about hang, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/running.rst)?
3. If your question is about docker, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/docker.rst)?
4. Did you check if you question is answered in the [troubleshooting guide](https://github.com/horovod/horovod/blob/master/docs/troubleshooting.rst)?

**Your question:**
Please ask your question here.

**Comment by chongxiaoc** [comment]
Yes, check examples here: https://github.com/horovod/horovod/tree/master/examples/tensorflow
